vmware_aria_operations_integration_sdk/docker_wrapper.py

```diff
@@ -89,9 +90,15 @@ def init() -> DockerClient:
     except docker.errors.DockerException as e:
         if "ConnectionRefusedError" or "FileNotFoundError" or "CreateFile" in e.args[0]:
             logger.debug(e, exc_info=True)
+
+            if platform.system() == "Windows":
+                host_os_port_path = "C:\ProgramData\docker"
+            else:
+                host_os_port_path = "/var/run/docker.dock"
```

Comment:
Is the `host_os_port_path` part of the message applicable to all three of these errors?